File: app/src/main/java/me/jdvp/androidaspectexample/application/AppModule.java
package me.jdvp.androidaspectexample.application;

import dagger.Module;
import me.jdvp.androidaspectexample.viewmodel.impl.ViewModelModule;

/**
 * Module that includes app wide dependencies that are not ViewModels (or included by ViewModelModule)
 *
 * Created by jdvp on 6/19/18.
 */
@Module(
        includes = ViewModelModule.class
)
class AppModule {
}

File: app/src/main/java/me/jdvp/androidaspectexample/application/AppModule.kt
package me.jdvp.androidaspectexample.application

import dagger.Module
import me.jdvp.androidaspectexample.viewmodel.impl.ViewModelModule

/**
 * Module that includes app wide dependencies that are not ViewModels (or included by ViewModelModule)
 */
@Module(includes = [ViewModelModule::class])
internal class AppModule